tf-managed/README.md

```diff
@@ -0,0 +1,11 @@
+# Terraform-managed
+
+This directory contains services and assets managed via Terraform/Terragrunt.
+
+# Structure
+
+```
+├── common <- common code, shared between all modules (TODO maybe move it to modules?)
```

Comment:
I like this folder structure. The only change I would make is to rename `common` to something like `scripts`, to show more intention on the folder itself. Since the root directory is `tf-managed`, we can assume that modules refer to terraform modules and either grow the scripts folder, which is then split by script language (i.e. ruby). Just a thought!

Reply:
@joshdougall Will do, thanks for the review. The end goal is to get rid of this directory altogether and hide all those scripts behind a docker (and/or packer!) images.